src/main/java/com/minecolonies/coremod/entity/ai/citizen/builder/ConstructionTapeHelper.java

```diff
@@ -0,0 +1,268 @@
+     */
+    public static void removeConstructionTape(@NotNull AbstractBuilding building, @NotNull World world)
+    {
+        removeConstructionTape(new WorkOrderBuild(building, 1), world);
+    }
+
+    public static void removeConstructionTape(@NotNull WorkOrderBuild workOrder,@NotNull World world)
+    {
+        final StructureWrapper wrapper = new StructureWrapper(world, workOrder.getStructureName());
+        final BlockPos pos = workOrder.getBuildingLocation();
+        int tempRotation = 0;
+        final IBlockState Air = Blocks.AIR.getDefaultState();
+        if (workOrder.getRotation() == 0 && !(workOrder instanceof WorkOrderBuildDecoration))
+        {
+            final IBlockState blockState = world.getBlockState(pos);
+            if (blockState.getBlock() instanceof AbstractBlockHut)
+            {
+                tempRotation = BlockUtils.getRotationFromFacing(blockState.getValue(AbstractBlockHut.FACING));
+            }
+        }
+        else
+        {
+            tempRotation = workOrder.getRotation();
+        }
+        wrapper.rotate(tempRotation, world, workOrder.getBuildingLocation());
+        wrapper.setPosition(pos);
+        int x1 = wrapper.getPosition().getX() - wrapper.getOffset().getX() - 1;
+        int z1 = wrapper.getPosition().getZ() - wrapper.getOffset().getZ() - 1;
+        int x3 = wrapper.getPosition().getX() + (wrapper.getWidth() - wrapper.getOffset().getX());
+        int z3 = wrapper.getPosition().getZ() + (wrapper.getLength() - wrapper.getOffset().getZ());
+        if (x1 < x3)
+        {
+            for (int i = x1; i <= x3; i++)
+            {
+                final BlockPos Block1 = new BlockPos(i, 0, z1);
+                final BlockPos Block2 = new BlockPos(i, 0, z3);
+                removeTapeIfNecessary(world,Block1,ModBlocks.blockConstructionTape);
+                removeTapeIfNecessary(world,Block2,ModBlocks.blockConstructionTape);
+            }
+        }
+        else
+        {
+            for (int i = x3; i <= x1; i++)
+            {
+                final BlockPos Block1 = new BlockPos(i, 0, z1);
+                final BlockPos Block2 = new BlockPos(i, 0, z3);
+                removeTapeIfNecessary(world,Block1,ModBlocks.blockConstructionTape);
+                removeTapeIfNecessary(world,Block2,ModBlocks.blockConstructionTape);
+            }
+        }
+        if (z1 < z3)
+        {
+            for (int i = z1; i <= z3; i++)
+            {
+                final BlockPos Block1 = new BlockPos(x1, 0, i);
+                final BlockPos Block2 = new BlockPos(x3, 0, i);
+                removeTapeIfNecessary(world,Block1,ModBlocks.blockConstructionTape);
+                removeTapeIfNecessary(world,Block2,ModBlocks.blockConstructionTape);
+            }
+        }
+        else
+        {
+            for (int i = z3; i <= z1; i++)
+            {
+                final BlockPos Block1 = new BlockPos(x1, 0, i);
+                final BlockPos Block2 = new BlockPos(x3, 0, i);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00117)